tests/cli/workflow/nf_analysis/test_cli_metrics_deliver.py

```diff
@@ -6,23 +6,23 @@
 from _pytest.logging import LogCaptureFixture
 from click.testing import CliRunner
 
-from cg.cli.workflow.rnafusion.base import metrics_deliver
-from cg.constants import EXIT_SUCCESS
+from cg.cli.workflow.base import workflow as workflow_cli
+from cg.constants import EXIT_SUCCESS, Workflow
 from cg.models.cg_config import CGConfig
 
 LOG = logging.getLogger(__name__)
 
 
 @pytest.mark.parametrize(
-    "context",
-    ["rnafusion_context", "taxprofiler_context"],
+    "workflow",
+    [Workflow.RNAFUSION, Workflow.TAXPROFILER],
 )
-def test_metrics_deliver_without_options(cli_runner: CliRunner, context: CGConfig, request):
+def test_metrics_deliver_without_options(cli_runner: CliRunner, workflow: Workflow, request):
     """Test metrics-deliver for Taxprofiler and Rnafusion without options."""
-    context = request.getfixturevalue(context)
+    context: CGConfig = request.getfixturevalue(f"{workflow}_context")
 
     # WHEN dry running without anything specified
```

Comment:
```suggestion
    # WHEN invoking the command with only the dry-run parameter specified
```

Reply:
updated to `# WHEN invoking the command without additional parameters`